--- automysqlbackup-v3.0_rc2/automysqlbackup
+++ automysqlbackup-v3.0_rc2/automysqlbackup
@@ -2039,7 +2039,7 @@ method_backup () {
 	# -> finished information
 	echo "Total disk space used for backup storage..."
 	echo "Size - Location"
-	echo `du -hsH "${CONFIG_backup_dir}"`
+	echo `du -hs --si "${CONFIG_backup_dir}"`
 	echo
 	echo "======================================================================"
 	# <- finished information
